website/src/examples/dashboard/code.sample

```diff
@@ -0,0 +1,51 @@
+import Uppy, { debugLogger } from '@uppy/core'
+import Dashboard from '@uppy/dashboard'
+import RemoteSources from '@uppy/google-drive'
+import ImageEditor from '@uppy/image-editor'
+import Form from '@uppy/form'
+import Webcam from '@uppy/webcam'
+import Audio from '@uppy/audio'
+import ScreenCapture from '@uppy/screen-capture'
+import Tus from '@uppy/tus'
+import DropTarget from '@uppy/drop-target'
+import Compressor from '@uppy/compressor'
+
+import COMPANION from '../env.js'
```

Comment:
what is this?

Reply:
Companion urls? Can just be constants.

Reply:
I think we want a code snippet that you can copy paste, right? But `env.js` is something specific so better inline variables indeed.